ex1/ex1.c

```diff
@@ -16,13 +20,24 @@ int main(void) {
 
     if (pid == 0) {
         x = 7;
-        printf("%d\n", x);
+        printf("%d\n", x);      // x = 7
     } else {
-        printf("%d\n", x);
-        wait(NULL); // if this is commentted result will be different.
-        printf("%d\n", x);
+        printf("%d\n", x);      // x = 100
+        x = 10;
+        // wait(NULL); // if this is commentted result will be different.
```

Comment:
Because it tells the program to wait for the Child to finish. Which is what you want.